packages/core/useWebWorkerFn/lib/createWorkerBlobUrl.ts

```diff
@@ -1,3 +1,4 @@
+import * as isoworker from 'isoworker'
```

Comment:
The core package should be dependency-free. If this is required, then it's probably out-scope and should be implemented on userland, or as a separate package